mainブランチのテストで、VVPPデフォルトエンジンのときのテストがたまに落ちる

## 不具合の内容

こちらとかです。
https://github.com/VOICEVOX/voicevox/actions/runs/12572256072/job/35044046584

とりあえず把握はしているんだけど、なぜ落ちるのかわかっていない状況です。

### 現象・ログ

```log
Error: [16:10:41.909] [error] 7z STDERR: ERROR: C:\Users\runneradmin\Downloads\voicevox_engine-windows-directml-0.21.0.vvpp
Cannot open the file as archive

The process cannot access the file because it is being used by another process.
```
とあります。
なぜかダウンロードしたファイルにアクセスできない感じになってるかもです。
一応ダウンロード後にファイルcloseはしてるつもりなのですが･･･。
https://github.com/VOICEVOX/voicevox/blob/18998167de481ddcff3891c671781a280b28a8f1/src/backend/electron/engineAndVvppController.ts#L233

### 再現手順

ローカルでも同じエラーを見た記憶があります。

### 期待動作

常に成功する

## その他


そもそもflakyだけど２回目は成功してそう･････？

```log
  1 flaky
    [electron] › electron\example.spec.ts:57:9 › VVPPデフォルトエンジン › 起動したら「利用規約に関するお知らせ」が表示される ─────────
  1 passed (2.4m)
  1 error was not a part of any test, see above for details
Error:   1) [electron] › electron\example.spec.ts:57:9 › VVPPデフォルトエンジン › 起動したら「利用規約に関するお知らせ」が表示される ────────
    Test timeout of 60000ms exceeded.
Notice:   1 flaky
    [electron] › electron\example.spec.ts:57:9 › VVPPデフォルトエンジン › 起動したら「利用規約に関するお知らせ」が表示される ─────────
  1 passed (2.4m)
Error: Process completed with exit code 1.
```

--- a/src/backend/electron/engineAndVvppController.ts
+++ b/src/backend/electron/engineAndVvppController.ts
@@ -230,7 +230,13 @@ export class EngineAndVvppController {
               progress: (downloadedBytes / totalBytes) * 100,
             });
           }
+
+          // ファイルを確実に閉じる
+          const { promise, resolve, reject } = Promise.withResolvers();
+          fileStream.on("close", resolve);
+          fileStream.on("error", reject);
           fileStream.close();
+          await promise;
 
           downloadedPaths.push(downloadPath);
           log.info(`Downloaded ${name} to ${downloadPath}`);

